.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: Autolabeler
+
+on:
+  # pull_request event is required for autolabeler
+  pull_request:
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:
```

Comment:
The `workflow_dispatch` won't be useful here because the auto-labeler needs to be used in a `pull_request` event. Please remove this trigger.

Reply:
```suggestion
```